ee/server/service/maintained_apps.go

```diff
@@ -139,19 +139,19 @@ func (svc *Service) AddFleetMaintainedApp(
 	return nil
 }
 
-func (svc *Service) ListFleetMaintainedApps(ctx context.Context, teamID uint, opts fleet.ListOptions) ([]fleet.MaintainedApp, *fleet.PaginationMetadata, error) {
+func (svc *Service) ListFleetMaintainedApps(ctx context.Context, teamID uint, opts fleet.ListOptions) ([]fleet.MaintainedApp, int, *fleet.PaginationMetadata, error) {
```

Comment:
Could the count be returned in the `TotalCount` field of the `PaginationMetadata` instead of adding another returned value? Or is this a different count?

Reply:
yep, I've updated so this is include in the `PaginationMetadata` data